src/pages/yourrides/offer/editOptions.story.tsx

```diff
@@ -0,0 +1,103 @@
+import Stepper, { StepperDisplay } from 'stepper'
+import Divider from 'divider'
+import ItemCheckbox from 'itemCheckbox'
+import ItemInfo from 'itemInfo'
+import Textarea from 'textarea'
+import ComfortIcon from 'icon/comfortIcon'
+import LadyIcon from 'icon/ladyIcon'
+import LightningIcon from 'icon/lightningIcon'
+import Button, { ButtonStatus } from 'button'
+
+const stories = storiesOf('Pages|Your rides/Offer/Edit/Options', module)
+
+type EditOptionsState = Readonly<{
+  isComfort: boolean
+  isLadyOnly: boolean
+  isInstantBooking: boolean
+  buttonStatus: ButtonStatus
+}>
+
+class EditOptions extends Component<{}, EditOptionsState> {
+  state: EditOptionsState = {
+    isComfort: false,
+    isLadyOnly: false,
+    isInstantBooking: false,
+    buttonStatus: ButtonStatus.PRIMARY,
+  }
+
+  save = (): void => {
+    this.setState({ buttonStatus: ButtonStatus.LOADING })
+
+    setTimeout(() => this.setState({ buttonStatus: ButtonStatus.CHECKED }), 1000)
+    setTimeout(() => this.setState({ buttonStatus: ButtonStatus.PRIMARY }), 3000)
+  }
+
+  updateOption = (name: string): void => {
+    this.setState((prevState) => ({ [name]: !prevState[name] }))
+  }
+
+  render = (): JSX.Element => (
+    <MainContent>
+      <Content>
+        <Section>
+          <SubHeader>Passengers options</SubHeader>
+
+          <div style={{ display: 'flex' }}>
+            <TextTitle style={{ flex: 1, lineHeight: '48px' }}>Number of passengers</TextTitle>
+            <div style={{ position: 'relative', right: '-14px' }}>
+              <Stepper
+                name="stepper"
+                value={1}
+                display={StepperDisplay.SMALL}
+                increaseLabel="Add a seat"
+                decreaseLabel="Remove a seat"
+              />
+            </div>
+          </div>
+
+          <Divider />
+
+          <ItemCheckbox
+            onChange={() => this.updateOption('isComfort')}
+            leftAddon={<ComfortIcon />}
+            labelTitle="Max. 2 in the back seats"
+            checked={this.state.isComfort}
+          />
+          <ItemCheckbox
+            onChange={() => this.updateOption('isLadyOnly')}
+            leftAddon={<LadyIcon />}
+            labelTitle="Ladies only"
+            checked={this.state.isLadyOnly}
+          />
+          <ItemCheckbox
+            onChange={() => this.updateOption('isInstantBooking')}
+            leftAddon={<LightningIcon />}
+            labelTitle="Instant booking"
+            checked={this.state.isInstantBooking}
+          />
+
+          <Divider />
+          <ItemInfo mainTitle="Additional details" />
+          <Textarea
+            placeholder="Flexible about where and when to meet? Not taking the motorway? Got limited space in your boot? Keep passengers in the loop."
+            onChange={() => {}}
+          />
+        </Section>
+      </Content>
+      <BottomContent style={{ display: 'flex', justifyContent: 'center', padding: '16px' }}>
```

Comment:
Do we have a "just center this button" behavior somewhere?
Also, do the new layout system adds de padding by itself on the button?

Reply:
@PerrineBoissieres Do we always display a centered button with 16px padding in the `BottomSection`? If so, we can add them to `BottomContent`...

Reply:
We don't have the "just center this button" in kirk yet.
We have the SubmitButton in kairos but it should be moved (ideally) to kirk.

Reply:
BottomContent as well as Content should not add any paddings, that's not their role.
Button can be used in many other components, so as well it should have its own margins or else it will mess up with a lot of cases.
Ideal solution would be to move the SubmitButton component to Kirk, yep, so it handles proper spacing and position.

Reply:
I'll create a ticket for this

Reply:
BBC-7944